src/site/content/en/blog/a11y-tips-for-web-dev/index.md

```diff
@@ -425,10 +413,9 @@ provide labels and values for each slice
 so users who have visual impairments can understand the information
 even if they can't tell where the slices begin and end:
 
-<figure class="w-figure">
-  <img src="./pie-chart.png" alt="A pie chart with labels and values to ensure accessibility.">
-  <figcaption class="w-figcaption">An accessible pie chart. (From the <a href="https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Informational_Graphic_Contrast_(Minimum)" rel="noopener">W3C Web Accessibility Initiative</a>.)</figcaption>
-</figure>
+<!--lint disable no-literal-urls-->
+{% Img src="image/admin/4DlbNfCX7AYYrPSpQg4L.png", alt="A pie chart with labels and values to ensure accessibility.", width="500", height="442", caption="An accessible pie chart. (From the <a href=\"https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Informational_Graphic_Contrast_(Minimum)\" rel=\"noopener\">W3C Web Accessibility Initiative</a>.)" %}
```

Comment:
I was surprised that the `<a>` in the caption still worked. That's cool that we can still support that.